### DISCUSSION TITLE
NameError: name 'git' is not defined

### DISCUSSION THREAD

**Discussion by dmbaker** [newren/git-filter-repo#354] [Q&A]
I installed the app via scoop, but when I run 

git-filter-repo --analyze

I get NameError: name 'git' is not defined.

Thanks, David

**Comment by AbbyCottontail** [comment]
Same, seems to be an issue with Python 3.10.4...

**Reply by AbbyCottontail** [reply]
Based on a suggestion I saw else where, I tried to downgrade to Python 3.8.8 and that still didn't work. So the issues isn't there.

I tried installing with scoop and I still had this issue. Hours lost trying to get this to work, I haven't given up and I found a workaround. 

See #360 for suggested work around @dmbaker
